From e227506c1977403e34e6b155af1bd5001ba4c02a Mon Sep 17 00:00:00 2001
Message-ID: <e227506c1977403e34e6b155af1bd5001ba4c02a.1708289807.git.sharlatanus@gmail.com>
In-Reply-To: <b6807bf457aa3debcd60c49908ebd1b147744e20.1708289807.git.sharlatanus@gmail.com>
References: <b6807bf457aa3debcd60c49908ebd1b147744e20.1708289807.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Fri, 16 Feb 2024 23:29:54 +0000
Subject: [PATCH v2 03/10] gnu: go-github-com-alecthomas-chroma: Remove bundled
 files.

* gnu/packages/golang-xyz.scm (go-github-com-alecthomas-chroma)
[source]: Remove git submodules and generated files.

Change-Id: Idafee9723b837495084aa0cd2fb454a953ff6233
---
 gnu/packages/golang-xyz.scm | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/gnu/packages/golang-xyz.scm b/gnu/packages/golang-xyz.scm
index 662341bf93..60ba4c0da4 100644
--- a/gnu/packages/golang-xyz.scm
+++ b/gnu/packages/golang-xyz.scm
@@ -88,8 +88,12 @@ (define-public go-github-com-alecthomas-chroma
              (commit (string-append "v" version))))
        (file-name (git-file-name name version))
        (sha256
-        (base32 "0hjzb61m5lzx95xss82wil9s8f9hbw1zb3jj73ljfwkq5lqk76zq"))))
+        (base32 "0hjzb61m5lzx95xss82wil9s8f9hbw1zb3jj73ljfwkq5lqk76zq"))
+       (modules '((guix build utils)))
+       ;; Delete git submodules and generated files by Hermit.
+       (snippet '(delete-file-recursively "bin"))))
     (build-system go-build-system)
+    ;; TODO: Build cmd/chroma and cmd/chromad commands.
     (arguments
      `(#:import-path "github.com/alecthomas/chroma"))
     (native-inputs
-- 
2.41.0

